.github/workflows/merge-python-requirements-upgrade-prs.yml

```diff
@@ -1,12 +1,27 @@
-# This file contains a workflow which will be run on schedule every day, and
+# This file contains a workflow which will be run on schedule every day, 
+# or can be call from other workflows and
 # will collect all the eligible (PRs which are labelled "Ready to merge" by our 
-# script) python upgrade PRs from both edx and openedx orgs.
+# script) python upgrade PRs from openedx or edx org.
 # After fetching all PRs, it will check if all the checks passing on the PR, if yes
 # then it will merge that PR, otherwise it will fail.
+# Note:
+# When workflow triggered from this repo(under openedx) will merge only openedx PRs 
+# and will merge edx orgs' PRs only when it will be called from the edx/.github repo
```

Comment:
Why is there edx.org specific logic in this openedx file?

Reply:
It is a reusable workflow that will be from the `edx/.gtihub` repo. We made it this way to keep the logic in one place so whenever a change is required, we need only one file to change.